controller/authController.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response, Application } from "express";
+import { Login } from "../model/login";
+import { Register } from "../model/register";
+import { log } from "console";
+const authService = require("../service/authService");
+const registerValidator = require("../validator/registrationValidator")
+
+module.exports = function(app: Application) {
+    app.get("/login", async (req:Request, res: Response) => {
+        res.render("login");
+    })
+
+    app.post("/login", async (req:Request, res:Response) => {
+        let data: Login = req.body;
+
+        try {
+            req.session.token = await authService.login(data)
+
+            // console.log(req.session.token);
+
+            res.redirect("/register");
```

Comment:
On a successful login you redirect back to the register screen. Would probably be better to redirect to a home view, even if it doesn't exist